### DISCUSSION TITLE
Omniverse PhysX vcpkg port now available

### DISCUSSION THREAD

**Discussion by marcodiiga** [NVIDIA-Omniverse/PhysX#162] [General]
```bash
$ vcpkg install physx
$ cd hello_physx/build # cmake based project
$ cmake -DCMAKE_BUILD_TYPE=Release .. && make
$ ./hello_physx
Starting PhysX up..
GPU acceleration is available
PhysX set up
Shutting down..
$
```

In an effort to alleviate the burden of setting up Omniverse PhysX compiler/linker options, presets and dependency management, I wrote a vcpkg port for Omniverse PhysX 5: [https://omni-101.github.io/omniverse/2023/07/07/omniverse-physx-vcpkg-port/](https://omni-101.github.io/omniverse/2023/07/07/omniverse-physx-vcpkg-port/)

Hopefully this might help people getting up-to-speed in evaluating, working, managing PhysX as a dependency in their projects for the OV supported platforms. The port just got merged in the official vcpkg repo.

Please note that **this is not an official NVIDIA maintained port** and should be considered as such (you are free to contribute/fix/extend this within the rules and contribution guidelines of NVIDIA PhysX and Microsoft vcpkg).

Hope this helps 🙂

**Comment by MasterDrake** [comment]
I guess a Blast port is not expected, right ?
